toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1739,6 +1739,9 @@ void MainWindow::defineActions() {
   QAction *mergeLevelsAction =
       createMenuXsheetAction(MI_MergeColumns, tr("&Merge Levels"), "");
   mergeLevelsAction->setIcon(QIcon(":Resources/merge.svg"));
+  QAction *mergeLevelsWOGroupsAction =
+      createMenuXsheetAction(MI_MergeColumnsWOGroups, tr("&Merge Levels Without Groups"), "");
+  mergeLevelsWOGroupsAction->setIcon(QIcon(":Resources/merge.svg"));
```

Comment:
You are using the same icon that the original Merge Levels command uses.

This could be confusing when added to tool bars as they won't know which one it is at a glance. I'd keep the new command's icon blank for now.

Reply:
I agree. I'll remove the icon.